zokrates_book/src/zokrates_js.md

```diff
@@ -60,3 +51,57 @@ initialize().then((zokratesProvider) => {
     const verifier = zokratesProvider.exportSolidityVerifier(keypair.vk, "v1");
 });
 ```
+
+## API
+
+##### initialize()
+Loads binding wasm module and returns a promise with ZoKrates provider.
+
+Returns: `Promise<ZoKratesProvider>`
+
+##### compile(source, location, resolveCallback[, config])
+Compiles source code into ZoKrates internal representation of arithmetic circuits.
+
+Parameters:
+* `source` - Source code to compile
+* `location` - Root location of the module which is being compiled
+* `resolveCallback` - User-defined callback used to resolve imports
+* `config` - Compilation config
+
+Returns: `CompilationArtifacts`
+
+##### computeWitness(artifacts, args)
+Computes a valid assignment of the variables, which include the results of the computation.
+
+Parameters:
+* `artifacts` - Compilation artifacts
+* `args` - Array of arguments (eg. `["1", "2", true]`)
+
+Returns: `ComputationResult`
+
+##### setup(program)
+Generates a trusted setup for the compiled program.
+
+Parameters:
+* `program` - Compiled program
+
+Returns: `SetupKeypair`
+
+##### exportSolidityVerifier(verificationKey, abi)
+Generates a Solidity contract which contains the generated verification key and a public function to verify a solution to the compiled program.
+
+Parameters:
+* `verificationKey` - Verification key from the setup keypair
+* `abi` - Abi version (`"v1"` | `"v2"`)
+
+Returns: `string`
+
+##### generateProof(program, witness, provingKey)
+Generates a proof for a computation of the compiled program.
+
+Parameters:
+* `program` - Compiled program
+* `witness` - Witness (valid assignment of the variables)
```

Comment:
It should be made clear that this is the output of the `computeWitness` step, but now that result is `ComputationResult` and here `witness`

Reply:
I left a comment for more clarity and added some examples to the computation step. Let me know if you have a better suggestion for this case